### PR TITLE
Add doc to README.

### LINKED ISSUE #4: Add doc to README

### PR DESCRIPTION
**Pull request by heuermh**
Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,134 @@ Examples for the [Disq](https://github.com/disq-bio/disq) library.
...
+SparkConf conf = new SparkConf()
+  .setAppName("Java Disq Example")
+  .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+  .set("spark.kryo.registrator", "org.disq_bio.disq.serializer.DisqKryoRegistrator")
+  .set("spark.kryo.referenceTracking", "true");
+
+JavaSparkContext jsc = new JavaSparkContext(new SparkContext(conf));
+```
+
+Then an [`JavaRDD`](https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.api.java.JavaRDD) is read using the
```
Comment (tomwhite) [inline README.md]: _a_ `JavaRDD`, not _an_ `JavaRDD`

File: README.md [hunk 2]
```diff
@@ -5,3 +5,134 @@ Examples for the [Disq](https://github.com/disq-bio/disq) library.
...
+[Disq APIs](https://www.javadoc.io/doc/org.disq-bio/disq)
+
+```java
+HtsjdkReadsRddStorage htsjdkReadsRddStorage = HtsjdkReadsRddStorage.makeDefault(jsc);
+HtsjdkReadsRdd htsjdkReadsRdd = htsjdkReadsRddStorage.read(args[0]);
```
Comment (tomwhite) [inline README.md]: Maybe say `args[0]` is a file path.

File: README.md [hunk 3]
```diff
@@ -5,3 +5,134 @@ Examples for the [Disq](https://github.com/disq-bio/disq) library.
...
+
+JavaRDD<SAMRecord> reads = htsjdkReadsRdd.getReads();
+```
+
+and analysis is performed via the [Spark JavaRDD APIs](https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.api.java.JavaRDD).
+
+
+### Running Java examples
+
+These examples can be run on the command line via `spark-submit`, e.g.
```
Comment (tomwhite) [inline README.md]: Doesn't the project need to be built first?

**Comment by heuermh** [comment]
Thanks, @tomwhite!  Pushed a commit to address your review comments.

**Comment by heuermh** [comment]
Thank you, @tomwhite

**Pull request merged**